code/modules/clothing/accessories/holster.dm

```diff
@@ -124,7 +124,7 @@
 	holster_verb_name = "Holster (Handgun)"
 
 /obj/item/clothing/accessory/holster/handgun/can_holster(obj/item/weapon/W)
-	if(!isgun(W) && !isbanana(W) && !istype(W, /obj/item/weapon/reagent_containers/food/snacks/grown/bluespacebanana))
+	if(!isgun(W) && !isbanana(W) && !istype(W, /obj/item/weapon/reagent_containers/food/snacks/grown/bluespacebanana) && !istype(W, /obj/item/weapon/gun/projectile/banana))
```

Comment:
Given that banana guns are `/obj/item/weapon/gun/projectile/banana`, wouldn't the `isgun()` check already allow them to be put into holsters?
Did you actually verify that they could not currently be put into holsters?

Reply:
THAT
is true.
I was reported that they didn't fit in the knife holsters like normal bananas so I assumed.
I'll just remove that.